src/tools/pbs_python.c

```diff
@@ -1092,15 +1092,18 @@ pbs_python_populate_svrattrl_from_file(char *input_file,
 	char *pc, *pc1, *pc2, *pc3, *pc4;
 	char *in_data = NULL;
 	char *tmp_data = NULL;
-	long int curpos;
 	long int endpos;
-	size_t   in_data_sz;
+	int in_data_sz;
 	char *data_value;
 	size_t ll;
 	FILE *fp = NULL;
 	char *p;
 	int   vn_obj_len = strlen(EVENT_VNODELIST_OBJECT);
 	int   job_obj_len = strlen(EVENT_JOBLIST_OBJECT);
+	int   b_triple_quotes = 0;
+	int   e_triple_quotes = 0;
+	char  buf_data[STRBUF];
+	int   buf_data_sz = STRBUF;
```

Comment:
This code looks very similar to get_hook_results() in mom_hook_func.c. Same comments apply.

Reply:
I took out buf_data_sz.